src/GraphQL/Types/GraphTypesLookup.cs

```diff
@@ -62,13 +72,10 @@ private void CheckSealed()
             IEnumerable<IGraphType> types,
             IEnumerable<DirectiveGraphType> directives,
             Func<Type, IGraphType> resolveType,
-            IFieldNameConverter fieldNameConverter,
+            INameConverter nameConverter,
             bool seal = false)
         {
-            var lookup = new GraphTypesLookup
-            {
-                FieldNameConverter = fieldNameConverter ?? CamelCaseFieldNameConverter.Instance
-            };
+            var lookup = nameConverter != null ? new GraphTypesLookup(nameConverter) : new GraphTypesLookup();
```

Comment:
Please invert this condition.

Reply:
Changed to the following:
```suggestion
            var lookup = nameConverter == null ? new GraphTypesLookup() : new GraphTypesLookup(nameConverter);
```
If that's not what you meant, lmk.

Reply:
This is it.